### PR TITLE
OSDOCS-8807: Small update to firewalls

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-8807](https://issues.redhat.com//browse/OSDOCS-8807): Small update to firewalls

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8807?focusedId=23653231&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23653231

Link to docs preview:
https://69442--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 1]
```diff
@@ -108,6 +108,10 @@ This section provides the necessary details that enable you to control egress tr
 |443
 |The `registry.access.redhat.com` and `https://registry.redhat.io` sites redirect through `catalog.redhat.com`.
 
+|`*.cloudfront.net`
+|443
+|Allows for CloudFront support.
```
Comment (arendej) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:113]: change to:
"Used by ROSA for its STS implementation with managed OIDC configuration."
or similar.

**Review by arendej — CHANGES_REQUESTED** [review]
see line 111-114, I left a comment

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**